Tools/Algorithms/psatd_pml.ipynb

```diff
@@ -276,8 +284,7 @@
     "    diff = X_t.diff(t).diff(t).subs(t, tn).subs(om, c * knorm).expand() - d2X_dt2\n",
     "    diff.simplify()\n",
     "    if diff != zeros(DD, 1):\n",
-    "        print(rf\"Could Not Verify Time Integration for {str(X)}\")\n",
-    "        display(diff)\n",
+    "        raise ValueError(rf\"Could not verify time integration for {str(X)}\")\n",
```

Comment:
We now fail the CI check with this kind of exceptions, to make sure that the notebook continues to be correct.